app/rubin-2025/sign-in-form.jsx

```diff
@@ -79,15 +86,17 @@ class SignInForm extends React.Component {
             </div>
           )}
 
-          {this.state.error && (
-            <div className="form-help error">
-              {this.state.error.message.match(/invalid(.+)password/i) ? (
-                <Translate content="signInForm.incorrectDetails" />
-              ) : (
-                <span>{this.state.error.toString()}</span>
-              )}{' '}
-            </div>
-          )}
+          <div aria-live="polite">
+            {this.state.error && (
+              <div id="sign-in-form-error-message" className="form-help error">
+                {this.state.error.message.match(/invalid(.+)password/i) ? (
+                  <Translate content="signInForm.incorrectDetails" />
+                ) : (
+                  <span>{this.state.error.toString()}</span>
+                )}{' '}
+              </div>
+            )}
+          </div>
 
           {this.state.busy ? (
             <LoadingIndicator />
```

Comment:
Using a live region to announce the loading state, as it changes, might make sense. Equally, it might also add extra noise to the page for a blind user.